Docs: new version of landing page for Alerting (#51166)
grafana/grafana: docs/sources/alerting/_index.md

aliases:
  - /docs/grafana/latest/alerting/
  - /docs/grafana/latest/alerting/unified-alerting/alerting/
  - /docs/grafana/latest/about-alerting
title: Alerting
weight: 114
---

# Grafana Alerting

> Grafana Alerting allows you to learn about problems in your systems moments after they occur. Create, manage, and take action on your alerts in a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.

Reviewer: Maybe worth adding an additional sentence to underline the important of the "any datasource"?

```suggestion
Grafana Alerting allows you to learn about problems in your systems moments after they occur. Create, manage, and take action on your alerts in a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.

Grafana Alerting works with queries and expressions from any existing data source — no matter where your data is stored.
```

> Depending on your use case, you can use Grafana Alerting for Grafana OSS, Grafana Enterprise, or Grafana Cloud. With Grafana OSS, you can create Grafana managed alerts or if your data already lives in Grafana Mimir or Grafana Loki, you can run alert expressions closer to your data and at massive scale, all managed by the Grafana UI.

Reviewer: Grafana Alerting is available for for Grafana OSS, Grafana Enterprise, or Grafana Cloud. With Mimir and Loki alert rules you can run alert expressions closer to your data and at massive scale, all managed by the Grafana UI you are already familiar with.
```suggestion
Grafana Alerting is available for for Grafana OSS, Grafana Enterprise, or Grafana Cloud. With Mimir and Loki alert rules you can run alert expressions closer to your data and at massive scale, all managed by the Grafana UI you are already familiar with.
```
